app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            diff = (self.clean_power - car.clean_mark)
+            return round(
+                (
+                    (
+                        car.comfort_class * diff * self.average_rating
+                    ) / self.distance_from_city_center
+                ), 1
+            )
+        return 0
```

Comment:
Don't agree 0 is float(small tip u can return 0.0 or use Union from typing)
But more important - u return 0 which is never used in code, because only implementation of code in your code is when CarWashStation.clean_power > Car.clean_mark, so this return is redundant